Source/Core/Core/IOS/FS/FileSystemProxy.cpp

```diff
@@ -211,7 +254,9 @@ u64 FSDevice::EstimateTicksForReadWrite(const Handle& handle, const ReadWriteReq
       const u32 start = offset - m_cache_chain_index * CLUSTER_DATA_SIZE;
       copy_length = std::min<u32>(CLUSTER_DATA_SIZE - start, count);
       // FS essentially just does a memcpy from/to an internal buffer.
-      ticks += 0.63f * copy_length + (request.command == IPC_CMD_WRITE ? 1000 : 0);
+      ticks += GetFSModuleMemcpyTbTicks(m_ios.GetVersion(), copy_length);
+      if (is_write)
+        ticks += 1000;
```

Comment:
Would it be the time to give this number a name? Guessing its some sort of cache flushing since its only on write.

Reply:
Done, cached writes seem to take slightly longer because of the free cluster check